examples/c/ex_col_store.c

```diff
@@ -0,0 +1,555 @@
+#include <test_util.h>
+
+#define N_DATA 100
+
+static const char *home;
+
+typedef struct {
+    char day[5];
+    uint16_t hour;
+    uint8_t temp;
+    uint8_t humidity;
+    uint16_t pressure;
+    uint8_t wind;
+    uint8_t feels_like_temp;
+    uint16_t loc_lat;
+    uint16_t loc_long;
+    char country[5];
+} WEATHER;
+
+uint8_t celsius_to_fahrenheit(uint8_t temp_in_celcius);
+void update_celsius_to_fahrenheit(WT_SESSION *session);
+void print_temp_column(WT_SESSION *session);
+void print_all_columns(WT_SESSION *session);
+void chance_of_rain(WT_SESSION *session);
+void generate_data(WEATHER *w_array);
+void remove_country(WT_SESSION *session);
+void search_temperature(WT_SESSION *session);
+void average_data(WT_SESSION *session);
+bool find_min_temp(WT_SESSION *session, uint16_t start_time, uint16_t end_time, int *result);
+bool find_max_temp(WT_SESSION *session, uint16_t start_time, uint16_t end_time, int *result);
+
+void
```

Comment:
Let's make these functions static, same for functions below. Also, the convention is to add a comment explaining the function if it is not static.